spec/factories/in_person_enrollments.rb

```diff
@@ -46,5 +46,9 @@
     trait :enhanced_ipp do
       sponsor_id { IdentityConfig.store.usps_eipp_sponsor_id }
     end
+
+    trait :enhanced_ipp do
+      sponsor_id { IdentityConfig.store.usps_eipp_sponsor_id }
+    end
```

Comment:
This dup should be deleted too- likely caused by merge conflict

Reply:
Ugh, this file wasn't flagged as conflicting during the merge conflict.  Thank you for catching that.